### PR TITLE
feature/inverted-oppi

### PR DESCRIPTION
**Pull request by Zackyist**
Adds three new functionalities to the bot:

### The inverted Oppi definition command
- Uses the format "¿¿ something"
- Prints out an inverted and reversed version of the Oppi definition

### The inverted random Oppi command
- Uses the format "¿¡"
- Prints out an inverted and reversed version of a random Oppi definition

### The /jokotai command
- Uses the format "/jokotai"
- Prints out a celebratory singsong message
- Prints out the Oppi definition of either "kruuna" or "klaava", chosen by random

### CODE REVIEW

File: oppija.py [hunk 1]
```diff
@@ -34,20 +45,73 @@ def opisCountHandler(self, bot, update, args=''):
...
+    def invertStringList(self, list):
+        # Reference table for the Unicode chars: http://www.upsidedowntext.com/unicode
+        chars_standard = 'abcdefghijklmnopqrstuvwxyzåäö'
+        chars_inverted = 'ɐqɔpǝɟbɥıɾʞןɯuodbɹsʇnʌʍxʎzɐɐo'
+
+        chars_standard += '_,;.?!/\\\'<>(){}[]`&'
+        chars_inverted += '‾\'؛˙¿¡/\\,><)(}{][,⅋'
+
+        chars_standard += 'ABCDEFGHIJKLMNOPQRSTUVWXYZÅÄÖ'
+        chars_inverted += '∀qϽᗡƎℲƃHIſʞ˥WNOԀὉᴚS⊥∩ΛMXʎZ∀∀O'
```
Comment (Cadiac) [inline oppija.py]: Have you checked that these chars render correctly on mobile devices?

File: oppija.py [hunk 2]
```diff
@@ -34,20 +45,73 @@ def opisCountHandler(self, bot, update, args=''):
...
+            inverted_question = re.match(r"^(\¿\¿)\s(\S+)$", msg.text)
             if question:
                 self.defineTerm(bot, update, question)
+            elif inverted_question:
+                self.defineTerm(bot, update, inverted_question, True)
+
             # Matches message "?!"
             elif re.match(r"^(\?\!)$", msg.text):
                 self.randomOppiHandler(bot, update)
+
```
Comment (Cadiac) [inline oppija.py]: Shouldn't this be ¿¡ ?

File: oppija.py (hunk 1 above)
Comment (Zackyist) [reply oppija.py]: Yeah, they are standard Unicode chars. I tested them in the standard Telegram client and they work as intended. I would presume they also work in Telegram X.

File: oppija.py (hunk 2 above)
Comment (Zackyist) [reply oppija.py]: No, the order is intentional. If you make a long Oppi, use this command, flip your screen upside down, and you'll see why. ;)

**Pull request merged**